src/github/workflows/deploy.yml

```diff
@@ -22,28 +27,18 @@ jobs:
         access_token: ${{ github.token }}
 
   build:
-    if: ${{ always() }}
+    if: ${{ !cancelled() }}
     needs: auto-cancel
     strategy:
       fail-fast: false
       matrix:
         # Andreas, 2022-10-18
         # We stick to '-latest' virtual environments here in the sense of
         # "most canonical", since this is the deploy action.
-        # As of today, this is [windows-2022, macOS-11, ubuntu-20.04].
-        os: [windows-latest, macos-latest]
+        # As of today (2024-02-02), this is [windows-2022, macOS-11, ubuntu-22.04].
```

Comment:
macOS-12, I'd think

Reply:
Perhaps it is better to refer to https://github.com/actions/runner-images#available-images...?